prac_05/word_counter.py

```diff
@@ -0,0 +1,25 @@
+"""
+CP1404/CP5632 Practical
+Count the occurrences of words in a given string.
+"""
+
+# text = "this is a collection of words of nice words this is a fun thing it is"
+text = input("Enter your text: ")
+words = text.split()
+found_words = {}
+for word in words:
+    try:
+        count = found_words[word]
```

Comment:
perhaps use a more descriptive word for count. seems a bit generic but I imagine for a larger program you would use a more specific name.